addons/website_profile/static/src/js/website_profile.js

```diff
@@ -42,6 +43,38 @@ sAnimations.registry.websiteProfile = sAnimations.Class.extend({
     },
 });
 
+sAnimations.registry.websiteProfileEditor = sAnimations.Class.extend({
+    selector: '.o_profile_editor',
+    /**
+     * @override
+     */
+    start: function () {
+        var self = this;
+        var $textarea = this.$('textarea.load_editor');
+        var $submit = this.$('.o_submit_btn');
+        if (!$textarea.length || !$submit.length) {
```

Comment:
That condition is useless, this is always the case and if not the code should break (not to dothing). You can remove this.